webpack/components/extensions/HostDetails/DetailsTabCards/HwPropertiesCard.js

```diff
@@ -14,8 +13,13 @@ import CardTemplate from 'foremanReact/components/HostDetails/Templates/CardItem
 import { TranslatedPlural } from '../../../Table/components/TranslatedPlural';
 import { hostIsNotRegistered } from '../hostDetailsHelpers';
 
-const HostDisks = ({ totalDisks }) => {
-  if (!totalDisks) return null;
+const HostDisks = ({ blockDevices }) => {
+  if (!blockDevices) return null;
+  // blockDevices fact will look like this by default 'sr0,sda' and increment like sdb etc
+  const disks = blockDevices.split(',');
+  // We are filtering out the CDROM drive that gets added by default a lot of the time
+  const totalDisks = disks.filter(e => !e.startsWith('sr')).length;
+  if (totalDisks.length === 0) return null;
```

Comment:
With

```js
const totalDisks = disks.filter(e => !e.startsWith('sr')).length;
```

`totalDisks` will be a number, right? because we're taking the `length` of the filtered array

So on the next line, why are we again checking `totalDisks.length`?  

```js
const totalDisks = 0
undefined
totalDisks.length
undefined
totalDisks.length === 0
false
```

Reply:
@jeremylenz Updated

```javascript
const totalDisks = 0
undefined
if (!totalDisks) console.log('hi')
VM247:1 hi
```

Went with this approach